README.md

```diff
@@ -1 +1,81 @@
-# ruby-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Ruby Capstone Project
+
+- OOP school library 
+
+## Project Description 
+
+In this project, you will create a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files but you will also prepare a database with tables structure analogical to your program's class structure.
+
+- Add new book, music album or game.
+- List all created and saved items
+- test all classes
+
+## Clone repo to run the program and decode the message!
+
+## Built With
+- Ruby
+- irb
+- CMD with Ruby
+- rspec
+- sql
+-postgres
+
+## Getting Started
+
+To get a copy of this project running on your local computer, do the followings:
+
+### Prerequisites
+- Must have a working computer
+- A Web-Browser installed (Chrome - Recommended)
+
+### Setup
+- Install a text Editor. For example [_VS_](https://visualstudio.microsoft.com/)
+- Clone this project repository on your local computer. [link](https://github.com/petushka1/ruby-capstone)
+- Open the project directory with _Text Editor_
+
+### Install
+
+```
+    bundler install
+```
+add on top of main.rb
+```
+    #!/usr/bin/env ruby
+```
+run
+```
+    chmod 755 main.rb
+```
+
+
+Follow all steps to install [_Ruby_](https://rubyinstaller.org/) 
+
+
+## Authors
+
+👤 **Nadya**
+
+- GitHub: [@petushka1](https://github.com/petushka1)
+- LinkedIn: [LinkedIn](http://www.linkedin.com/in/nadyalevina)
+
```

Comment:
- [ ] Since this is a group project, it will be nice update your readme file with information pertaining to other group member